iroha_data_model/src/lib.rs

```diff
@@ -180,6 +186,19 @@ impl Value {
             SignatureCheckCondition(s) => s.0.len(),
         }
     }
+
+    /// Number of underneath expressions.
+    #[cfg(feature = "roles")]
+    pub fn len(&self) -> usize {
+        use Value::*;
+
+        match self {
+            U32(_) | Id(_) | PublicKey(_) | Bool(_) | Parameter(_) | Identifiable(_)
+            | String(_) | TransactionValue(_) | PermissionToken(_) | Role(_) => 1,
```

Comment:
I guess you can put `#[cfg(...)]` on variant itself, am i wrong?

Reply:
Just seems like a code duplication to me

Reply:
I think this is not possible. [Example](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=061996ac7cf9b236d0b9b28f8c61fde7).

Reply:
It works this way:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=d599bba0120a3ef44f5222d2361509e7